src/NexusMods.Sdk/Tracking/Events.cs

```diff
@@ -252,6 +252,10 @@ public readonly record struct DurationValue(long Milliseconds)
     {
         public static implicit operator DurationValue(long value) => new(value);
         public static implicit operator DurationValue(TimeSpan value) => new((long)value.TotalMilliseconds);
-        public static implicit operator DurationValue(Stopwatch value) => new(value.ElapsedMilliseconds);
+        public static implicit operator DurationValue(Stopwatch value)
+        {
+            value.Stop();
+            return new DurationValue(value.ElapsedMilliseconds);
+        }
```

Comment:
Shouldn't the stop happen on the caller side? This seems like it could easily be delayed, making the communicated elapsed time wrong

Reply:
`Stop` is a no-op when called on an already stopped Stopwatch. The caller could stop the watch themselves but since the precision is only on a millisecond basis, the time loss is often `0` or negligible, especially since this implicit operator gets called first before the actual method gets called.